doc/user/compatibility.md

```diff
@@ -212,9 +212,6 @@ but these may behave differently or be removed in future versions.
 
 ## Features not yet supported on the SVM
 
-* Signal handlers
-* Finalizers
-* Setting the process name (for example `$0 =`)
 * Java interop
 
 Most of these features will silently not work, in order to let as much as
```

Comment:
This text needs to be adapted as well.

Reply:
I think as it's rendered as an HTML list, it still works. But I can update it.

Reply:
The text after, "while potentially missing signals, leaking resources, etc." is no longer appropriate and misleading.
But I think we don't need the entire paragraph.

Reply:
My mistake. I thought you were referring to "Most of these features...", since that's all GitHub rendered. I'll clean it up.